### PR TITLE
[IMP] udes_stock: Use same logic for unassigning batches in logout an…

### PR DESCRIPTION
**Pull request by joe-gb**
drop_off_picked and unassign_user_batch now use the same logic and function so that the functionalities now match.

### CODE REVIEW

**Comment by parisiale** [comment]
Thanks for this!

Sorry for being pedantic, but I'm afraid your intent is not clear in your commit message; you're describing the changes, that are clear by simply looking at the diff, but not your reasons and objectives. I would assume you're looking for consistency, but what's the real benefit?. 
Describing the intent is fundamental for the reviewer, but most importantly for the future reader of the commit tree. And that should be focus of the description.

Also, you're not separating the commit title (gigantic) from the description. I suggest to have a look at https://chris.beams.io/posts/git-commit/.

File: addons/udes_stock/models/stock_picking_batch.py [hunk 1]
```diff
@@ -506,16 +503,19 @@ def get_user_batches(self):
...
     def unassign_user_batches(self):
-        """ Removes pickings from all batches assigned to the user and cancels the batch
+        """ Get batches for user and unassign them
         """
         # Get user batches
-        batches = self.get_user_batches()
+        self.get_user_batches().unassign()
 
+    def unassign(self):
+        """ Removes incomplete pickings and removes responsible user from batches
```
Comment (parisiale) [inline addons/udes_stock/models/stock_picking_batch.py]: Docstring looks good, but would it make sense to have two methods instead, one to unassign another to remove the incomplete pickings, so to improve naming and functionality clarity?

File: addons/udes_stock/models/stock_picking_batch.py [hunk 2]
```diff
@@ -506,16 +503,19 @@ def get_user_batches(self):
         return batches
 
     def unassign_user_batches(self):
-        """ Removes pickings from all batches assigned to the user and cancels the batch
+        """ Get batches for user and unassign them
```
Comment (parisiale) [inline addons/udes_stock/models/stock_picking_batch.py]: You're also removing the incomplete pickings.

File: addons/udes_stock/models/stock_picking_batch.py (hunk 1 above)
Comment (joe-gb) [reply addons/udes_stock/models/stock_picking_batch.py]: Probably not, it's one method - the act of "unassigning" the batch is to follow the logic of the "unassign" function as defined: It's to unassign the user AND the incomplete pickings IF it's ephemeral

Comment (parisiale) [reply addons/udes_stock/models/stock_picking_batch.py]: 👍

File: addons/udes_stock/models/stock_picking_batch.py (hunk 2 above)
Comment (joe-gb) [reply addons/udes_stock/models/stock_picking_batch.py]: yes! unassigning a batch is to do just that - if it's ephemeral

**Comment by joe-gb** [comment]
> Thanks for this!
> 
> Sorry for being pedantic, but I'm afraid your intent is not clear in your commit message; you're describing the changes, that are clear by simply looking at the diff, but not your reasons and objectives. I would assume you're looking for consistency, but what's the real benefit?
> Describing the intent is fundamental for the reviewer and for the future reader of the commit tree. And that should be focus of the description.
> 
> Also, you're not separating the commit title (gigantic) from the description. I suggest to have a look at https://chris.beams.io/posts/git-commit/.

Tidied up the commit message

**Comment by parisiale** [comment]
👍

**Pull request merged**